### PR TITLE
feat(root): add code examples for page header using slots

### PR DESCRIPTION
**Pull request by zxh722**
## Summary of the changes

Code example added in for changing slots within the page header, changing the typography of a page header semantically.

## Related issue

#632 

## Checklist

- [x] I have [manually accessibility tested](https://design.sis.gov.uk/accessibility/testing/manual-testing) any changes, if relevant.
- [x] I have raised this pull request against the [develop branch](https://github.com/mi6/ic-design-system/tree/develop)

### CODE REVIEW

**Review by GCHQ-Developer-299 — DISMISSED** [review]

**Review by jd239 — CHANGES_REQUESTED** [review]
Just a single comment :)

**Review by GCHQ-Developer-299 — APPROVED** [review]

File: src/content/structured/components/page-header/code.mdx [hunk 1]
```diff
@@ -480,3 +481,70 @@ export const withReactRouter = [
...
+      <ic-typography variant="h1" slot="heading">Latte recipe</ic-typography>
+      <ic-typography slot="subheading">A Latte is a popular Italian coffee, made with espresso, steamed milk and a thin layer of foam.</ic-typography>
+      <ic-chip slot="heading-adornment" label="BETA" size="large"></ic-chip>
+      <ic-breadcrumb-group slot="breadcrumbs">
+        <ic-breadcrumb current="true" page-title="Drinks" href="#"></ic-breadcrumb>
+        <ic-breadcrumb page-title="Coffees" href="#"></ic-breadcrumb>
+        <ic-breadcrumb page-title="Latte" href="#"></ic-breadcrumb>
+      </ic-breadcrumb-group>
+      <ic-text-field slot="input" placeholder="Search for ingredients…" label="Input" hide-label="true" />
+</ic-page-header>`,
```
Comment (jd239) [inline src/content/structured/components/page-header/code.mdx:500]: So the indentation is still not quite right. I think line 491-499 needs to be outdented to remove the empty spaces.

Comment (jd239) [reply src/content/structured/components/page-header/code.mdx:500]: Same with the React snippet:
image

**Review by jd239 — CHANGES_REQUESTED** [review]
Quick formatting issues

**Review by jd239 — APPROVED** [review]

**Review by GCHQ-Developer-299 — APPROVED** [review]

**Pull request merged**